kubectl-from-path/README.md

```diff
@@ -0,0 +1,12 @@
+# Kubectl
+
+This task uses kubectl to do a specific action (e.g create, apply, delete) from a certain path
+[`kubectl`](https://kubernetes.io/en/docs/reference/kubectl/kubectl/) for this.
+
+Note that you'll probably want to use this in combination with a cloned GitHub repository.
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/kubectl-from-path/kubectl.yaml
```

Comment:
Once the task is added in the task directory could you please update the path as well